Makefile

```diff
@@ -19,3 +19,8 @@ install:
 	@echo -e "$(TARGET_COLOR)Running install$(NO_COLOR)"
 	@npm clean-install --prefer-offline --cache .npm
 	@npm list
+
+eslint:
+	@echo -e "$(TARGET_COLOR)Running eslint $$(npx eslint --version)$(NO_COLOR)"
+	@npx eslint .; \
+	echo "Passed"
```

Comment:
i beleive this should be ok, since at the beginning of the makefiule we set:

```
SHELL := /bin/bash -euo pipefail
```

make should therefore fail on the first encountered error